src/libm/e_exp.c

```diff
@@ -98,6 +98,13 @@ P3   =  6.61375632143793436117e-05, /* 0x3F11566A, 0xAF25DE2C */
 P4   = -1.65339022054652515390e-06, /* 0xBEBBBD41, 0xC5D26BF1 */
 P5   =  4.13813679705723846039e-08; /* 0x3E663769, 0x72BEA4D0 */
 
+union {
+    Uint64 u64;
+    double d;
+} inf_union = {
+    0x7ff0000000000000
+};
```

Comment:
This should perhaps be replaced with:
```
static SDL_FORCE_INLINE double get_inf(void) {
    return (double)(1.0f / 0.f);
}
```
and wrap it with `#pragma`'s to silence warnings.

Reply:
Bad idea: [C2124 is an error and impossible to disabe](https://learn.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/compiler-error-c2124)